web-admin/src/routes/[organization]/[project]/[dashboard]/+page.svelte

```diff
@@ -74,11 +70,10 @@
   }
 
   $: dashboard = useDashboard(instanceId, dashboardName);
-  $: isDashboardOK = $dashboard.isSuccess;
   $: isDashboardNotFound =
     $dashboard.isError &&
     ($dashboard.error as QueryError)?.response?.status === 404;
-  $: isDashboardErrored = !$dashboard.data?.metricsView?.state?.validSpec;
+  $: isDashboardErrored = !!$dashboard.data?.meta?.reconcileError;
```

Comment:
Why are we going back to not supporting dashboards that were valid once? @begelundmuller did anything change in the backend?

Reply:
Oh good catch, that's my bad. I was trying to make this code easier to read. This intricacy slipped by me. I've reverted this and instead added a comment to explain it.